source/developers-guide/rest-api/api-resource-user/index.md

```diff
@@ -0,0 +1,207 @@
+
+These examples assume you are using the provided **[demo API client](/developers-guide/rest-api/#using-the-rest-api-in-your-own-application)**.
+
+
+## Get a list of users
+If you want to get multiple user at once, you can call the /users endpoint.
+
+*Available arguments:*
+| Argument            | Type         | Required            | Description                                                             |
+|---------------------|--------------|---------------------|-------------------------------------------------------------------------|
+| limit               | int          |                     | Max. number of returned data sets                                       |
+| start               | int          |                     | Offset (ideal for batch processing, when working with large data sets)  |
+| sort                | string array |                     | ORDER BY clause                                                         |
+| filter              | string array |                     | Filter properties by expressions                                        |
+
+
+*Example code:*
+```php
+$client->get('users', [
+    'limit' => 10, // limit the number of users to 10
+    'start' => 1, // skip the first one
+    'sort' => [
+        ['property' => 'username', 'direction' => 'DESC']
+    ]
+    
+]);
+```
+
+*Example output:*
+```json
+{
+    "total": 2,
+    "data": [
+        {
+            "id": 1,
+            "roleId": 1,
+            "localeId": 1,
+            "username": "demo",
+            "password": "$2y$10$d7s.jETNFL1lZL3OzY7PneVWGk16aRZR9iuGyGHnw3X5EzssJ304W",
+            "encoder": "bcrypt",
+            "apiKey": "DaxN5BdfmcyglZMEwopy8Z46yADINhqViztSfcvI",
+            "sessionId": "f0ga0998i8b86aaahfjhjva760",
+            "lastLogin": "2017-11-01T11:10:24+0100",
+            "name": "Demo user",
+            "email": "demo@example.com",
+            "active": 1,
+            "failedLogins": 0,
+            "lockedUntil": "2010-01-01T00:00:00+0100",
+            "extendedEditor": false,
+            "disabledCache": false
+        },
+        {
+            "id": 2,
+            "roleId": 6,
+            "localeId": 1,
+            "username": "test",
+            "password": "$2y$10$SuT6CVqrHsnZbG29kqsVq.DYXhx.JbF4X13bLlkxOb9dl/a4OIQym",
+            "encoder": "bcrypt",
+            "apiKey": "ohwrzHP70iwUkBdzPEx6iUfSc3sLrHZ7678dy3Ie",
+            "sessionId": "",
+            "lastLogin": "2017-11-01T10:48:41+0100",
+            "name": "asdasd",
+            "email": "asdsad.asd@asd.de",
+            "active": 1,
+            "failedLogins": 0,
+            "lockedUntil": "2017-11-01T10:48:41+0100",
+            "extendedEditor": false,
+            "disabledCache": false
+        }
+    ],
+    "success": true
+}
+```
+
+Attention: The properties apiKey, sessionId and password are missing,
+the API user neither has the update nor the create privilege.
```

Comment:
is there an `if` missing before `the API`?

Reply:
You are right! Fixed it.